mesos-docker/run/run.sh

```diff
@@ -524,59 +544,59 @@ function show_help {
 
   Options:
 
+  --cpu-th the percentage of the host memory to use for slaves. Default: 0.5.
```

Comment:
I think you meant cpu instead of memory here. Also --cpu-th sounds like a weird flag name? how about --cpu-host-percentage?